src/platforms/react-native/performance/included-instrumentation.mdx

```diff
@@ -152,7 +246,9 @@ const App = () => {
 
 #### Custom Instrumentation
 
-```js
+To create a custom routing instrumentation, look at how the `RoutingInstrumentation` class [is implemented](https://github.com/getsentry/sentry-react-native/blob/master/src/js/tracing/routingInstrumentation.ts). If you need an example of how to implement routing instrumentation, review the code of the existing official instrumentation such as [the one for React Navigation V4](https://github.com/getsentry/sentry-react-native/blob/master/src/js/tracing/reactnavigationv4.ts)
```

Comment:
please use a pinned link with the hash, so if we ever rename the file or move around, it wont break the link